doc/source/index.rst

```diff
@@ -255,6 +255,131 @@ the same start and end times and coverage, just different data.
 Historically this was created to collect SEVIRI segments, which has some
 impact on the configuration.
 
+In the segment gatherer YAML configuration, the user can define one or
+more patterns that are collected.  The following top level variables
+may be defined:
+
+patterns
+    Mapping of pattern names to pattern definitions.  Each category definition is itself a mapping
+    that must contain the key ``pattern`` and may contain the keys ``critical_files``, ``wanted_files``,
+    ``all_files``, ``is_critical_set``, and ``variable_tags``.  When ``patterns``
+    is not defined, the segment gatherer will not do anything useful.
+
+    pattern
+        Defines the pattern used to parse filenames obtained from incoming
+        posttroll messages.  The string follows trollsift syntax.
+        The labels ``channel_name`` and ``segment`` have special meaning.
+        They must be defined as strings in the pattern, because the segment
+        gatherer formats the filename pattern /after/ converting numeric
+        segments or segment ranges to strings.
```

Comment:
I mean that `{segment:4d}` will not work, because `0000` gets changed to `????`, which is invalid for `:4d`.  At least I think that's how I remember it.  I have adapted the formulation in 0f099f8.